include/dxc/HLSL/DxilContainerWriter.h

```diff
@@ -0,0 +1,55 @@
+///////////////////////////////////////////////////////////////////////////////
+//                                                                           //
+// DxilContainerWriter.h                                                     //
```

Comment:
Let's move all container stuff to DxilContainer lib, since you don't want it in DXIL lib, and it probably doesn't belong under HLSL either.